ligand/templates/ligand_statistics.html

```diff
@@ -146,10 +146,12 @@
 
 {% if render != "bias" %}
 <h4>Number of unique ligands per receptor class</h4>
+<table style="width:80% !important" class="table table-stripped">
 {% else %}
-<h4>Number of unique ligands bias per receptor class</h4>
+<p><h4>Biased ligand counts</h4>
+<h6>Ligands with a bias factor equal or greater to 2</h6></p>
```

Comment:
I think the code for applying this new cutoff for the bias factor is missing on the backend.